src/o365/spo/commands/page/clientsidepages.ts

```diff
@@ -850,16 +854,24 @@ export class ClientSideWebpart extends ClientSidePart {
     this.order = index;
 
     // will form the value of the data-sp-webpartdata attribute
-    const data = {
+    let data = {
```

Comment:
`data` can stay a `const` since further on we're only modifying its properties and not the whole object